bpy_speckle/operators/commit.py

```diff
@@ -11,19 +11,19 @@
 
 class DeleteCommit(bpy.types.Operator):
     """
-    Deletes the selected commit from the selected stream.
+    Permanently deletes the selected version from the selected model.
```

Comment:
is the change from `project` (stream) to `model` intentional?

Reply:
It is, I was trying to align with the new api. Where versions are a property of a model, and not of a stream.
But... perhaps we could discuss this more as a team, and align on our naming here.